app/views/users/index.html.haml

```diff
@@ -10,10 +11,13 @@
 #result
   %h3= text(".head")
   %p= text(".instruct")
+
   - if @users.any?
-    = render "search/results_table", order_for: true, users: @users
+    = render "search/results_table",
+      order_for: current_facility.try(:single_facility?),
```

Comment:
The `@users` here are `UserPresenter`-wrapped. I'm going to merge in #865 and retest.